openSUSE/daps
# Fix screen linebreaks

Searches for `<screen>` tags in DocBook files and replace misplaced linebreaks.

Tested scenarios:

* Linebreak between `<screen>` and text
* Linebreak between `<screen>` and entity.
* Linebreak between `<screen>` and `<prompt>`, `<replaceable>`

### Testing

1. Update/clone this repo.
1. Switch to the `fix-screen` branch.
1. Change directory to your doc repository.
1. Create an alias and replace the `PATH_TO_YOUR_DAPS_REPO` with the correct value:

       $ alias fix-screen="PATH_TO_YOUR_DAPS_REPO/python-scripts/fix-screen/bin/fix-screen.py

1. Run:

       $ fix-screen xml/YOUR_XML_FILE.xml

1. Check the diff.
